packages/dates/src/tests/apply-time-zone-offset.test.ts

```diff
@@ -57,5 +57,14 @@ describe('applyTimeZoneOffset()', () => {
         applyTimeZoneOffset(date, 'Australia/Sydney', 'UTC'),
       ).toStrictEqual(expected);
     });
+
+    it('applies extra dst offset for days where DST is applied or reverted', () => {
+      const date = new Date('2020-11-01T04:00:00.000Z');
+      const expected = new Date('2020-11-01T06:00:00.000Z');
+
+      expect(
+        applyTimeZoneOffset(date, 'America/Denver', undefined),
```

Comment:
Does this test partially rely on the timezone of the test runner? If so, do we need to explicitly set it with jest?

Reply:
no, I don't believe so, I am however going to add an explicit second timezone to this function call. 👀